.github/dependabot.yml

```diff
@@ -4,21 +4,16 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
-      time: "04:00" # UTC
-    #labels:
-    #  - "domain: deps"
     commit-message:
-      prefix: "chore(deps)"
+      prefix: "security"
     reviewers:
       - "bruceg"
-    open-pull-requests-limit: 10
+    open-pull-requests-limit: 0
```

Comment:
Did you mean to set this to `0`?  That appears to disable it per https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit

Reply:
Ah, nvm, I see the note there now:

> This option has no impact on security updates, which have a separate, internal limit of ten open pull requests.